automation/createWorkspaces.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -e
+
+
+for i in {1..501}; do
+    curl -X POST --header 'Content-Type: application/json' --header 'Accept: application/json' --header 'Authorization:
+    Bearer ya29.GltaBjfPdBbkVXbuYsOiCIcguA4zjyVZJGZ5nqb9q5Gt3qgEHMjolrPSdPkmtHJB8l6tfC5cqJ3Rus3iEf8wNzUifh4VmYZWb47gFvgAxC31FFcmAClaTPTELgSD' -d '{ \
```

Comment:
hardcoded bearer token

did you intend for this file to be in here?

Reply:
Shit this file is not suppose to be promoted, it supposed to reside on my machine, not sure how it slipped through, I'll delete it